controllers/ytsaurus_local_test.go

```diff
@@ -56,6 +58,11 @@ func TestYtsaurusFromScratch(t *testing.T) {
 	defer h.Stop()
 
 	ytsaurusResource := testutil.BuildMinimalYtsaurus(namespace, ytsaurusName)
+	// FIXME(khlebnikov): This test is broken by design.
```

Comment:
What happens if min ready not set to zero? It never becomes ready? What prevents test to work with default nil value?

Reply:
This test runs without controller bundle - without statefulset controller nobody creates pods.

Reply:
ah, yes, no point of waiting running state here